src/indigo/less/new-ui/index.less

```diff
@@ -12,18 +19,38 @@
 @new-ui-black: #333333;
 @new-ui-white: #ffffff;
 @new-ui-border: #e6e6e7;
+@new-ui-border-ligth: rgba(162, 165, 169, 0.15);
+@new-ui-shadow: rgba(86, 86, 90, 0.05);
+@new-ui-text-grey: #ababad;
+@new-ui-svg-hover: #272837;
 
-body.new-ui {
+body {
   font-family: 'Roboto', sans-serif;
-  padding: 0;
+  margin: 0;
   min-height: 100%;
 
   @import 'helpers';
-  @import 'basis';
-  @import 'buttons';
-  @import 'login';
-  @import 'alerts';
-  @import 'current-user';
-  @import 'projects-list';
-  @import 'modal';
+  @import 'main';
```

Comment:
@natocTo asi len poznamocka, ze toto ovplyvni vsetko ... ale ak sa nic nerozbilo, asi som s tym ok :)

Reply:
Jo to potřebujeme. Tam je právě úprava toho main-contentu apod. Takové základní věci na grafiku aby seděla. Plus asi nějaké drobnosti (strong tag vždy 500 font-weight) ale myslím že by to mělo být ok.

Reply:
Jo ale vlastně nový vzhled. Ale současný jsem měl puštěný a vypadal ok.

Reply:
Myslel som to skor tak, ze je to mimo tych "new-ui" a "new-ui-menu" tried, cize to ovplyvni aj sucasny design.

Reply:
Ale ak sa nic nerozbilo, nevadi to. Tiez to este pozeram.

Reply:
Jo vše vypadá ok. Jako asi by se nic nestalo kdyby se to přesunulo pod nové UI jen. Ale toto ovlivní drobnosti jako bordery, shadow na main contentu apod.

Reply:
Přesunu to nakonec. Fakt asi nemá smysl to mít i pro staré UI, akorát pak bude více podobné to nepotřebujeme.

Reply:
Oka, hlavne nech o tom vies. ;) Mergnem to za par min.